docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,163 @@
+
+First Run
+=========
+
+Now you have installed MCDR, it's time to run it for the first time...?
+
+.. note::
+
+    We assume you have basic knowledge of managing a Minecraft server with one of the common server softwares. Which means, you should know:
+
+    * The Minecraft server software: what is it, how to configure it
+    * Start command of Minecraft server: what is it, what its arguments mean
+    * Console of the server: what it prints, commands you can send to it
+    * YAML and JSON files: what they are, how to edit them
+    
+    If you have not, Google is your friend
+
+Minecraft Server Software
+-------------------------
+
+Most common use of MCDR is to control a Minecraft server. In order to do that, you should have a **well-configured** Minecraft server software
+
+For example, this is a typical directory structure of vanilla Minecraft server:
+
+.. code-block:: text
+
+    server/
+    ├── libraries/
+    ├── logs/
+    ├── versions/
+    ├── world/
+    ├── banned-ips.json
+    ├── banned-players.json
+    ├── eula.txt
+    ├── ops.json
+    ├── server.jar
+    ├── server.properties
+    ├── usercache.json
+    └── whitelist.json
+
+With this command to start it:
+
+.. code-block:: bash
+
+    java -Xmx1024M -Xms1024M -jar server.jar nogui
```

Comment:
Use this, keep them synchronized

https://github.com/MCDReforged/MCDReforged/blob/8231d850241e12f849b80771c3aca62fae230475/mcdreforged/resources/default_config.yml#L29